variants/aws-dev/Cargo.toml

```diff
@@ -36,6 +36,7 @@ included-packages = [
     "strace",
     "tcpdump",
     "chrony-tools",
+    "xfsprogs",
```

Comment:
Looks good for now while working through next steps. Just to note, would be good to get this moved to be part of the `os` package once we are ready.

Reply:
Correct, for the moment, this is just to ensure it gets built at all. The next PR will be adding in the proper support for XFS in variants which will move this to the right place.

Reply:
I agree we should only include it in the variants that will be released with XFS. Existing variants should remain in ext4.